### PR TITLE
Update to powsybl dependencies 2023.0.1.

### PR DESCRIPTION
**Pull request by obrix**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: src/test/resources/contingencies/6noeuds.json [hunk 1]
```diff
@@ -1 +1 @@
-{"type":"default","version":"1.0","name":"contingencies","contingencies":[{"id":"SE_G","elements":[{"id":"SE_G","type":"GENERATOR"}]},{"id":"SE_L2","elements":[{"id":"SE_L2","type":"LOAD"}]},{"id":"SO_G2","elements":[{"id":"SO_G2","type":"GENERATOR"}]},{"id":"SO_G1","elements":[{"id":"SO_G1","type":"GENERATOR"}]},{"id":"N_G","elements":[{"id":"N_G","type":"GENERATOR"}]},{"id":"HVDC1","elements":[{"id":"HVDC1","type":"HVDC_LINE"}]},{"id":"SO_NO_1","elements":[{"id":"SO_NO_1","type":"BRANCH"}]},{"id":"SO_NO_2","elements":[{"id":"SO_NO_2","type":"BRANCH"}]}]}
\ No newline at end of file
+{"type":"default","version":"1.0","name":"contingencies","contingencies":[{"id":"SE_L2","elements":[{"id":"SE_L2","type":"LOAD"}]},{"id":"SO_NO_2","elements":[{"id":"SO_NO_2","type":"BRANCH"}]},{"id":"SO_NO_1","elements":[{"id":"SO_NO_1","type":"BRANCH"}]},{"id":"N_G","elements":[{"id":"N_G","type":"GENERATOR"}]},{"id":"SO_G2","elements":[{"id":"SO_G2","type":"GENERATOR"}]},{"id":"HVDC1","elements":[{"id":"HVDC1","type":"HVDC_LINE"}]},{"id":"SO_G1","elements":[{"id":"SO_G1","type":"GENERATOR"}]},{"id":"SE_G","elements":[{"id":"SE_G","type":"GENERATOR"}]}]}
```
Comment (annetill) [inline src/test/resources/contingencies/6noeuds.json]: Why the order has changed?

File: src/test/resources/loadflowreferences/loadflow_6n_standard.xiidm [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<iidm:network xmlns:iidm="http://www.powsybl.org/schema/iidm/1_8" xmlns:hopr="http://www.itesla_project.eu/schema/iidm/ext/hvdc_operator_active_power_range/1_0" xmlns:hapc="http://www.itesla_project.eu/schema/iidm/ext/hvdc_angle_droop_active_power_control/1_0" id="Reseau_test_Topase_avec_report_seuils_reduits" caseDate="2015-09-29T09:14:00.000+01:00" forecastDistance="0" sourceFormat="NF" minimumValidationLevel="STEADY_STATE_HYPOTHESIS">
+<iidm:network xmlns:iidm="http://www.powsybl.org/schema/iidm/1_9" xmlns:hopr="http://www.itesla_project.eu/schema/iidm/ext/hvdc_operator_active_power_range/1_0" xmlns:hapc="http://www.itesla_project.eu/schema/iidm/ext/hvdc_angle_droop_active_power_control/1_0" id="Reseau_test_Topase_avec_report_seuils_reduits" caseDate="2015-09-29T09:14:00.000+01:00" forecastDistance="0" sourceFormat="NF" minimumValidationLevel="STEADY_STATE_HYPOTHESIS">
```
Comment (annetill) [inline src/test/resources/loadflowreferences/loadflow_6n_standard.xiidm:2]: Why do we have to change this file? Only CGMES files are impacted by the new dependencies.

Comment (obrix) [reply src/test/resources/loadflowreferences/loadflow_6n_standard.xiidm:2]: It was regenerated with the latest version of powsybl the schema has been updated.

File: src/test/resources/contingencies/6noeuds.json (hunk 1 above)
Comment (obrix) [reply src/test/resources/contingencies/6noeuds.json]: There was in fact a random process regenerating the contingencies in the test case creator. I avoided using that to regenerate the reference as I was regenerating different tests cases. This is fixed.

**Pull request merged**